ajax/index.html

```diff
@@ -251,102 +252,121 @@ <h1 class="center">Asynchronous Technologies: AJAX/JSON</h1>
       </div><!--Requests and Responses-->
 
       <div class="slide normal">
-        <header><h1>1) Creating an XMLHttpRequest</h1></header>
+        <header><h1>Anatomy of an XMLHttpRequest</h1></header>
         <section class="content">
 <pre>
-var request; 
-try {
-  if (typeof ActiveXObject != 'undefined') {
-    request = new ActiveXObject('Microsoft.XMLHTTP');
-  } else if (window["XMLHttpRequest"]) {
-    request = new XMLHttpRequest();
-  }
-} catch (e) {
-  request = false;
-}
-if (!request ) alert("Error initializing XMLHttpRequest!");
+// instantiate a new request
+var request = new XMLHttpRequest(); 
+
+// add event listeners
+request.onload = function() = {
+
+  // transform a string into a usable object
+  console.log( JSON.parse( request.responseText ) );
+};;
+
+request.open('get', '/path/to/api', true);
+request.setRequestHeader('Content-type', 'application/json');
+request.send();
 </pre>
-          <p>IE implemented XMLHR differently than the other browsers, so you must use all this code to get it cross-browser-compatible.</p>
- 
-          <p>Most Javascript libraries/APIs offer wrapper functions for this.</p>
         </section>
-      </div><!--creating XMLHttpRequest-->
+      </div><!--Anatomy of an XMLHttpRequest-->
 
       <div class="slide normal">
-        <header><h1>2) Sending an XMLHttpRequest</h1></header>
+        <header><h1>XMLHttpRequest POST</h1></header>
         <section class="content">
+        <p>When using POST, you will want to also send along some data.</p>
 <pre>
-function sendRequest() { 
-  url="books.json";
-  request.open("GET", url, true);
-  request.onreadystatechange = processRequest;
-  request.send(null); 
+var data = {
+  kittenNames: ['Mary-Ann', 'Brenda'],
```

Comment:
i'm trying to figure out if this is one kitten or 2 or one kitten with 2 personalities.